CTS accessor category compfail 'Unresolved extern function '_Z22__spirv_ImageQuerySizeIDv3_i14ocl_image3d_roET_T0_'' with latest SYCL build and CUDA

**Describe the bug**
CTS accessor category compfail with CUDA backend on latest SYCL builds and latest CUDA version. See logs below:

```sh
FAILED: bin/test_accessor
: && clang++ -ffp-model=precise  -fsycl -fsycl-unnamed-lambda -Wno-deprecated-declarations  -Wl,-no-relax    -fsycl -fsycl-targets=nvptx64-nvidia-cuda-sycldevice -sycl-std=2020 -Xsycl-target-backend --cuda-gpu-arch=sm_50 tests/accessor/CMakeFiles/test_accessor_objects.dir/accessor_api_image_fp16.cpp.o tests/common/CMakeFiles/main_function_object.dir/main.cpp.o -o bin/test_accessor  -Wl,-rpath, util/libutil.a oclmath/liboclmath.a libOpenCL.so && :
clang++: warning: unknown CUDA version: cuda.h: CUDA_VERSION=11040.; assuming the latest supported version 10.1 [-Wunknown-cuda-version]
ptxas fatal   : Unresolved extern function '_Z22__spirv_ImageQuerySizeIDv3_i14ocl_image3d_roET_T0_'
llvm-foreach:
clang++: error: ptxas command failed with exit code 255 (use -v to see invocation)
clang version 14.0.0 (https://github.com/intel/llvm.git bf7eb4cacca8b47208abf534f002955c1e9e9ec5)
Target: x86_64-unknown-linux-gnu
Thread model: posix
InstalledDir: 
clang++: note: diagnostic msg: Error generating preprocessed source(s).
ninja: build stopped: cannot make progress due to previous errors.
```
**To Reproduce**
_Note_: Due to a recent uplift of minimal supported C++ standard, one should modify this line to require C++17.
1. Get SYCL with CUDA support ([GSG](https://intel.github.io/llvm-docs/GetStartedGuide.html)) and [SYCL-CTS](https://github.com/KhronosGroup/SYCL-CTS)
2. Build CTS tests
```sh
cd SYCL-CTS
mkdir build && cd build
cmake -GNinja  -DSYCL_IMPLEMENTATION=Intel_SYCL -DINTEL_SYCL_ROOT=<path to built sycl> -Dopencl_platform_name=nvidia -Dopencl_device_name=opencl_gpu -DCMAKE_BUILD_TYPE=Release -DINTEL_SYCL_FLAGS="-Xsycl-target-backend;--cuda-gpu-arch=sm_50" -DCMAKE_EXE_LINKER_FLAGS=" -Wl,-no-relax " -DINTEL_SYCL_TRIPLE=nvptx64-nvidia-cuda-sycldevice -DSYCL_CTS_ENABLE_OPENCL_INTEROP_TESTS=Off -DSYCL_CTS_ENABLE_DOUBLE_TESTS=On -DSYCL_CTS_ENABLE_HALF_TESTS=On ..
ninja test_accessor
```

**Environment (please complete the following information):**
OS version: Ubuntu 20.04.3 LTS
CUDA version:
```sh
nvidia-smi

+-----------------------------------------------------------------------------+
| NVIDIA-SMI 470.57.02    Driver Version: 470.57.02    CUDA Version: 11.4     |
|-------------------------------+----------------------+----------------------+
```

--- a/libclc/ptx-nvidiacl/libspirv/images/image.cl
+++ b/libclc/ptx-nvidiacl/libspirv/images/image.cl
@@ -973,7 +973,7 @@ _CLC_DEFINE_IMAGE_SAMPLED_READ_BUILTIN(half, DF16_, 3, int4, Dv4_i, float4)
 #undef _CLC_DEFINE_IMAGE_SAMPLED_READ_BUILTIN
 
 // Size Queries
-_CLC_DECL int _Z22__spirv_ImageQuerySizeIDv2_i14ocl_image1d_roET_T0_(
+_CLC_DECL int _Z22__spirv_ImageQuerySizeIDv1_i14ocl_image1d_roET_T0_(
     read_only image1d_t image) {
   return __nvvm_suq_width_1i(image);
 }
@@ -985,10 +985,10 @@ _CLC_DECL int2 _Z22__spirv_ImageQuerySizeIDv2_i14ocl_image2d_roET_T0_(
   return (int2)(width, height);
 }
 
-_CLC_DECL int4 _Z22__spirv_ImageQuerySizeIDv2_i14ocl_image3d_roET_T0_(
+_CLC_DECL int3 _Z22__spirv_ImageQuerySizeIDv3_i14ocl_image3d_roET_T0_(
     read_only image3d_t image) {
   int width = __nvvm_suq_width_3i(image);
   int height = __nvvm_suq_height_3i(image);
   int depth = __nvvm_suq_depth_3i(image);
-  return (int4)(width, height, depth, 0);
+  return (int3)(width, height, depth);
 }

